style.css

```diff
@@ -102,77 +107,171 @@ td{min-width: 8em; border: 1px solid black; height: 25em;}
-	display: -ms-flexbox;
-	display: flex;
-	-webkit-box-orient: vertical;
-	-webkit-box-direction: normal;
-	    -ms-flex-direction: column;
-	        flex-direction: column;
+.form{
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+        -ms-flex-direction: column;
+            flex-direction: column;
+
+    justify-content: center;
 }
 
-.short {
-	width: 2.5em;
+.form .short{
+    width: 40px;
+    margin: 5px;
+    position: relative;
+    bottom: 10px;
+    height: 20px;
 }
 
-.options > * {
-	margin: 1em 0 0 0;
+
+.dontfilldiv{
+    width: 100%;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    justify-content: center;
+    margin-bottom: 0;
 }
 
-button#add-musts {
-	width: 100%;
-	height: 2.5em;
+.dontfillbutton{
+    text-transform: none;
+    width: 7em;
+    padding: 2px;
+    font-size: 14px;
+    position: relative;
+    bottom: 5px;
+    height: 25px;
+    line-height: 1em;
 }
 
-button#make {
-	width: 100%;
-	height: 2.5em;
-	margin-top: 0.5em;
-	background-color: #ffd;
+.flashlight{
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    justify-content: center;
 }
 
-.courses {
-	background-color: #eee;
-	margin-top: 0.5em;
-	display: -webkit-box;
-	display: -ms-flexbox;
-	display: flex;
-	-webkit-box-orient: vertical;
-	-webkit-box-direction: normal;
-	    -ms-flex-direction: column;
-	        flex-direction: column;
+.flashlight > * {
+    margin: 2px;
+    height: 3em;
+    line-height: 1em;
+}
+
+.stuff{
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+        -ms-flex-direction: column;
+            flex-direction: column;
+    margin: 20px;
+    margin-top: 45px;
+    width: 350px;
+}
+
+/* Larger than Desktop HD */
+@media (max-width: 1264px) {
+    .stuff{
+        margin-top: 20px;
+    }
+}
+
+
+.isflexaround{
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    justify-content: space-around;
 }
 
+.fullwidth{
+    width: 100%;
+}
+
+.stuffbutton{
+    color: #fff;
+    background-color: #5F4C62;
+}
+
+.addcoursebutton{
+    height: 29px;
+    line-height: 1em;
+}
+
+.courses{
+    border: solid 1px #5F4C62;
+    border-radius: 4px;
+    background-color: #eee;
+    margin-top: 0.5em;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+        -ms-flex-direction: column;
+            flex-direction: column;
+    font-family: sans-serif;
+}
+
+.close {
+    float: right;
+    height: 2em;
+    width: 2em;
+    padding: 0;
+    line-height: 1em;
+}
+
+.toggle{
+    width: 6em;
+    height: 2em;
+    line-height: 1em;
+    padding: 0;
+    position: relative;
+    bottom: 5px;
+}
+
+/* COPY PASTE FROM LIBDUCK. PROPABLY RELATED WITH SOME THINGS*/
```

Comment:
I know this is not an actual comment but let's keep the CSS comment free too. anlayan anlasın anlamayan karpuz yesin aq